sdk-setup/src/mb2

```diff
@@ -1634,10 +1779,26 @@ if [[ $1 != --self-test ]]; then  ###  M A I N  EXECUTION BEGINS HERE  #########
 # If this is called with an empty argument, the default settings is restored.
 handle_pro_file_or_dir_option() {
     local pro_file_or_dir=$1
+    local cmakecache_file="CMakeCache.txt"
+    local read_from_cmakecache=0
 
     rm -f "$STATEDIR/src"
 
-    if [[ ! $pro_file_or_dir ]]; then
+    # Read the project source path from the CMakeCache file, if the path hasn't
+    # been passed as an argument and the CMakeCache exists.
```

Comment:
So what exactly are you suggesting?

Reply:
I would first consider disallowing to omit the source path on subsequent `mb2 cmake` calls, and if this turns not to be acceptable then consider extending `mb2 qmake` (and `mb2 build`?) so that it also allows to omit the source path on subsequent calls.